pyproject.toml

```diff
@@ -1,56 +1,93 @@
+description = "Python SDK and CLI Client for ServiceX"
+readme = "README.md"
+license = { text = "BSD-3-Clause" }  # SPDX short identifier
+requires-python = ">=3.8"
 authors = [
-    "Ben Galewsky <bengal1@illinois.edu>",
-    "Gordon Watts <gwatts@uw.edu>",
+    { name = "Ben Galewsky", email = "bengal1@illinois.edu" },
+    { name = "Gordon Watts", email = "gwatts@uw.edu" },
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
+    "qastle>=0.17",
+    "func_adl>=3.2.6",
+    "requests>=2.31",
+    "pydantic>=1.10,<2.0",  # FIXME: Adopt pydantic 2.0 API
+    "httpx>=0.24",
+    "miniopy-async>=1.15",
+    "tinydb>=4.7",
+    "google-auth>=2.17",
+    "typer[all]>=0.9.0",
+    "PyYAML>=6.0",
+    "types-PyYAML>=6.0",
+    "rich>=13.0.0",  # databinder
+    "nest-asyncio>=1.5.7",  # databinder # FIXME: nest-asyncio deprecated
+    "aiofile",  # compatible versions controlled through miniopy-async
+    "make-it-sync",  # compatible versions controlled through func_adl
 ]
-readme = "README.md"
-packages = [{ include = "servicex" }]
-
-[tool.poetry.scripts]
-servicex = "servicex.app.main:app"
-
-[tool.poetry.dependencies]
-python = "^3.8, < 3.12"
-qastle = "^0.16"
-func_adl = "^3.2.6"
-requests = "^2.31"
-pydantic = "^1.10"
-httpx = "^0.24"
-miniopy-async = "^1.15"
-tinydb = "^4.7"
-google-auth = "^2.17"
-typer = { extras = ["all"], version = "^0.9.0" }
-PyYAML = "^6.0"
-types-PyYAML = "^6.0"
-make_it_sync = "^1.0.0"
-
-pandas = { version = "^2.0.2", optional = true }
-pyarrow = { version = "^12.0.0", optional = true }
-fastparquet = { version = "^2023.4.0", optional = true }
-
-sphinx = { version = "^7.0.1", optional = true }
-furo = { version = "^2023.5.20", optional = true }
-jupyter = "^1.0.0"
-
-
-[tool.poetry.group.test.dependencies]
-pytest = "^7.2.0"
-pytest-mock = "^3.10.0"
-pytest-cov = "^4.0.0"
-flake8 = "^5.0.4"
-mypy = "^0.981"
-pytest-asyncio = "^0.21.0"
-asyncmock = "^0.4.2"
 
+[project.urls]
+Documentation = "https://github.com/ssl-hep/ServiceX_frontend"
+Homepage = "https://github.com/ssl-hep/ServiceX_frontend"
+"Issue Tracker" = "https://github.com/ssl-hep/ServiceX_frontend/issues"
+"Release Notes" = "https://github.com/ssl-hep/ServiceX_frontend/releases"
+"Releases" = "https://github.com/ssl-hep/ServiceX_frontend/releases"
+"Source Code" = "https://github.com/ssl-hep/ServiceX_frontend"
```

Comment:
As there are no public docs yet just list the GitHub for everything for the time being.